programmers/짝수와-홀수.js

```diff
@@ -0,0 +1,12 @@
+function solution(num) {
+    var number
```

Comment:
let 또한 var처럼 초기값을 넣어주지 않아도 됩니다.
```suggestion
    let number
```